### ISSUE TITLE
Found 0 attachments

### ISSUE THREAD

**Issue by mrchang0815** [nickrussler/email-to-pdf-converter#27]
Contents of eml-file:

```
...
Content-Type: multipart/alternative;
	boundary="Apple-Mail=_50C3F72A ..."
Mime-Version: 1.0 (Mac OS X Mail 13.4 \(3608.120.23.2.1\))
...
X-Mailer: Apple Mail (2.3608.120.23.2.1) 
...
--Apple-Mail=_2C37B240 ...
Content-Disposition: inline;
	filename="111111 XXXXXXX-XXXXXX.pdf"
Content-Type: application/pdf;
	x-unix-mode=0644;
	name="111111 XXXXXXX-XXXXXX.pdf"
Content-Transfer-Encoding: base64

JVBERi0xLjMK...
...SVFT0YK
--Apple-Mail=_2C37B240-...
Content-Transfer-Encoding: quoted-printable
Content-Type: text/html;
	charset=utf-8
...
```

The pdf is generated successfully, but the attachment is missing. From Thunderbird I can normally save the attachment. 

Started the coverter in debug-mode:

```
Mime Structure of blabla
-----------Mime Message-----------
> multipart/alternative
> |  text/plain
> |  multipart/mixed
> |  |  text/html
> |  |  application/pdf; inline
> |  |  text/html
----------------------------------
...
[MimeMessageConverter.convertToPdf()#319]: Start extracting attachments
Extract attachments to blablabla
[MimeMessageConverter.convertToPdf()#333]: Found 0 attachments
Conversion finished

```

Edit 28.09.2020, 18:01:
If I change the Content-Disposition from "inline" to "attachment", the attachment is extracted successfully. This observation may be helpful for you.

**Comment by nickrussler** [comment]
Since only images are embedded in the HTML as of now, it makes sense to extract any other attachments. Following [this](https://github.com/bbottema/simple-java-mail/issues/83) reasoning, extracting unreferenced images makes sense too.

**Comment by nickrussler** [comment]
Should be fixed in 2.4.0, please check it @mrchang0815

**Comment by mrchang0815** [comment]
Hello,

thank you very much for your efforts. I tested the modified version.  
The extraction of inline attachments works well now.

Perfect!

Regards,
"mrchang0815"
Zitat von Nick Russler <notifications@github.com>:

> Should be fixed in 2.4.0, please check it @mrchang0815
>
> --
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/nickrussler/email-to-pdf-converter/issues/27#issuecomment-703773199

-- 
Dominique Maier | Max-Pechstein-Straße 11 | 52076 Aachen | T.  
02408/704380 | F. 704381 | mailto: dominique@maier-online.net
